src/services/course.test.js

```diff
@@ -0,0 +1,275 @@
+import { Op } from 'sequelize';
+import { expect } from '@playwright/test';
```

Comment:
I don't think we want to import this here

Reply:
Good call.